api/v1/alerts_policy_types.go

```diff
@@ -75,27 +76,25 @@ func init() {
 	SchemeBuilder.Register(&AlertsPolicy{}, &AlertsPolicyList{})
 }
 
-func (in AlertsPolicySpec) APIAlertsPolicy() alerts.AlertsPolicy {
+func (in AlertsPolicySpec) APIAlertsPolicy() alerts.Policy {
```

Comment:
Should this be `alerts.AlertsPolicy`?  Is it even used?

Reply:
Good catch.  Updated.